CONTRIBUTING.md

```diff
@@ -1,4 +1,24 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
 ## Contributing to this Project
 
 OpenSearch is a community project that is built and maintained by people just like **you**.
 [This document](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md) explains how you can contribute to this and related projects.
+The brief summary below outlines contribution guidelines specific to the OpenSearch-Security Repo.
+
+## Triaging
+
+OpenSearch-Security seeks to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project.
+
+FAQ about OpenSearch-Security Triage Meetings:
+
+* Do I need to attend fo rmy issue to be addressed/triaged?
```

Comment:
The formatting on these doesn't look very clean.  Maybe we can reorganized answers with more prose and details so they can use more space


![image](https://user-images.githubusercontent.com/2754967/197642067-97af647e-f2f3-441d-a098-f705eab12261.png)